test/test_linalg.py

```diff
@@ -5269,7 +5269,7 @@ def test_householder_product_errors_and_warnings(self, device):
             with self.assertRaisesRegex(RuntimeError, "Expected all tensors to be on the same device"):
                 torch.linalg.householder_product(reflectors, tau)
 
-    @precisionOverride({torch.float32: 1e-4, torch.complex64: 1e-4})
+    @precisionOverride({torch.float32: 1e-2, torch.complex64: 1e-2})
```

Comment:
Yes. Note that this is some sort of integration test between all the LU-related functions. All the other functions (computing the LU decomposition) are much more stable than computing this inverse. Hence the looser tolerances.

And yes, tests were failing otherwise.